### ISSUE TITLE
Pre-provisioning error

### ISSUE THREAD

**Issue by ccjjmartin** [geerlingguy/drupal-vm#641]
## Background:
- I have backed up to a stable release 2.5.1 and everything is fine but on the latest commit in master (57fb88d) pre-provisioning fails with this error message:

```
TASK [Run configured pre-provision shell scripts.] *****************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: IndexError: list index out of range
fatal: [gw]: FAILED! => {"failed": true, "msg": "Unexpected failure during module execution.", "stdout": ""}
```

## Additional Information:
- My config.yml might have something out of date in it.  I can verify that it works with 2.5.1.  I am willing to try a different commit if that would help diagnose this problem.

**Comment by oxyc** [comment]
Would you mind posting your `config.yml` and checking which ansible version you are using? I'd like to see if I can reproduce it.

**Comment by oxyc** [comment]
Also to provide some more ansible debugging information could you create a `Vagrantfile.local` in the same directory as the regular `Vagrantfile` with this:

```rb
config.vm.provisioners[0].config.raw_arguments = ["-vvv"]
```

**Comment by joelpittet** [comment]
I ran into something similar.  Add this
`pre_provision_scripts: []`
next to
`post_provision_scripts: []` 

in your config.yml

**Comment by oxyc** [comment]
@joelpittet could you check which ansible version you have installed?

**Comment by oxyc** [comment]
Nevermind, just realized you fixed it. I was able to reproduce it by removing the variable. Thanks!

I'll try to get a proper fix for it if the variable isn't defined.

**Comment by ccjjmartin** [comment]
@joelpittet I can confirm that your fix works.

@oxyc I can confirm that your patch works. 👍

**Comment by geerlingguy** [comment]
Fixed via merged PR above #643
